.github/workflows/ci.yml

```diff
@@ -87,7 +87,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
 
       - name: Install Deno
-        if: matrix.job == 'lint'
+        if: matrix.job == 'lint' || matrix.job == 'test'
```

Comment:
Could changes to this file be done in a separate PR?

Reply:
#15098